components/integration/staging/base/kustomization.yaml

```diff
@@ -11,6 +11,11 @@ images:
   newName: quay.io/redhat-appstudio/integration-service
   newTag: 55b2bae85419df8cd8b9e633d212826a7b9567ee
 
+configMapGenerator:
+- name: console-url
+  literals:
+    - CONSOLE_URL="https://console.dev.redhat.com/preview/application-pipeline/ns/%s/pipelinerun/%s"
```

Comment:
as we discussed, can we keep consistency with PAC and use templating instead of format string?

https://github.com/redhat-appstudio/infra-deployments/blob/6141189cea62ccdd9827dd29f4fa3eddf92b73e6/components/pipeline-service/production/base/update-tekton-config-pac.yaml#L8

Reply:
Needed to make namespace > .Namespace and pr > .PipelineRunName to work properly.